### ISSUE TITLE
Custom Env Registration Example/Guide/Information Not Found

### ISSUE THREAD

**Issue by gradnerj** [DLR-RM/stable-baselines3#594]
### Question

I'm attempting to use RL-Zoo for hyperparameter tuning on a DDPG agent and custom environment, is there currently a guide or description of this process? 

### Additional context
 I've looked in the documentation on the stable-baselines3 and rl-baselines3-zoo repos and cannot seem to find any detailed information on registering a custom environment. I've looked here, <a>https://github.com/DLR-RM/stable-baselines3/blob/master/docs/guide/custom_env.rst</a> and the complete guide link returns 404, Page Not Found. The zoo repo's README does not provide details on registering other than informing you that you must if you want to use a custom env with zoo. 

# Additional Information
I've read through the issue <a>https://github.com/DLR-RM/stable-baselines3/issues/259</a> and have seen <a>https://github.com/DLR-RM/rl-baselines3-zoo/blob/master/utils/import_envs.py</a>. The link provided to a simple implementation provided in that thread also returns 404. 

### Checklist

- [x ] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/) 
- [ x] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo

**Comment by Miffyli** [comment]
Sounds like this issue should be opened in the rl-zoo repository.

**Comment by gradnerj** [comment]
Ah yes, my mistake. I’ll close this and move it there.

**Comment by araffin** [comment]
> Sounds like this issue should be opened in the rl-zoo repository.

well we need to update SB3 doc too, the link gives a 404 error...
@gradnerj best is to look at gym code to understand how env registration work.

**Comment by gradnerj** [comment]
> @gradnerj best is to look at gym code to understand how env registration work.

Is env registration absolutely required to use rl-zoo? Also, does registering an environment with gym make it available to the public? Thanks for the help.

**Comment by Miffyli** [comment]
> Is env registration absolutely required to use rl-zoo? Also, does registering an environment with gym make it available to the public? Thanks for the help.

Nope, it will not be public. "Env registering" simply means you create the environment following Gym API and then use the register function to give it a string-name (e.g. "Breakout-v0"), so you can use `gym.make("Breakout-v0")` to create that environment.

**Comment by gradnerj** [comment]
> Nope, it will not be public. "Env registering" simply means you create the environment following Gym API and then use the register function to give it a string-name (e.g. "Breakout-v0"), so you can use `gym.make("Breakout-v0")` to create that environment. This is the easiest way to use rl-zoo.

Great, makes sense! Thanks a bunch for the info.
